roles/openshift_master/tasks/registry_auth.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Check for credentials file for registry auth
+  stat:
+    path: "{{ oreg_auth_credentials_path }}"
+  when: oreg_auth_user is defined
+  register: master_oreg_auth_credentials_stat
+
+# Container images may need the registry credentials
+- name: Setup ro mount of /root/.docker for containerized hosts
```

Comment:
We need to bind this directory into the container so the container can pull images.  This may not be necessary on the master container services, I am not sure.  Previous details indicated we need these credentials on both masters and nodes.

What we're doing here:  Binding /var/lib/origin/.docker to /root/.docker.